script.js

```diff
@@ -0,0 +1,81 @@
+const logo = document.querySelector('.ham');
+const xButons = document.querySelector('.close-button');
+const menu = document.querySelector('#menu-pop');
+
+logo.addEventListener('click', () => {
+  menu.style.display = 'block';
+});
+
+xButons.addEventListener('click', () => {
+  menu.style.display = 'none';
+});
+
+document.querySelectorAll('.popupButtons').forEach((n) => n.addEventListener('click', () => {
+  menu.style.display = 'none';
+}));
+
+const attendees = [
+  {
+    image: 'images/stefano.jpeg',
+    name: 'Stefano Domenicalli',
+    role: 'President & CEO Formula1',
+    history: 'After joining Ferrari as a young man,he worked his way up to become Team Principal',
+  },
+
+  {
+    image: 'images/hamilton.jpeg',
+    name: 'Lewis Hamilton',
+    role: '103 Grand Prix Winner',
+    history: 'British racing driver currently competing in Formula One, driving for Mercedes.',
+  },
+
+  {
+    image: 'images/schumacher3.jpeg',
+    name: 'Michael Schumacher',
+    role: '91 Grand Prix Winner',
+    history: 'Former German racing driver who competed in Formula One for Jordan, Benetton, Ferrari, and Mercedes',
+  },
+
+  {
+    image: 'images/vettel2.jpeg',
+    name: 'Sebastian Vettel',
+    role: '53 Grand Prix Winner',
+    history: 'German racing driver who competed in Formula One from 2007 to 2022 for BMW Sauber, Toro Rosso, Red Bull, Ferrari and Aston Martin.',
+  },
```

Comment:
- [ ] [OPTIONAL] Good job adding dynamic speakers in the main page 👍 , It could be better if you implement the `see more` feature to improve the user experience 😄 

### Your design 
![Capture 1](https://github.com/MarkYegon7/Capstone-Project/assets/80458199/b5c94f8d-7267-41a8-99be-67e247b684d2)

### Cindly design

![Capture 1](https://github.com/MarkYegon7/Capstone-Project/assets/80458199/a37e196b-2dd9-488f-8ba5-bb1f3d7199a0)